Diabetes/Startup/APHProfileExtender.m

```diff
@@ -119,15 +135,94 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtAdjustedIndexPath:(N
   */
 - (void)navigationController:(UINavigationController *)navigationController didSelectRowAtIndexPath:(NSIndexPath *) __unused indexPath
 {
-    UIStoryboard *sbGlucoseLog = [UIStoryboard storyboardWithName:@"APHOnboarding" bundle:[NSBundle mainBundle]];
-    APHGlucoseLevelsMealTimesViewController *controller = [sbGlucoseLog instantiateViewControllerWithIdentifier:@"APHGlucoseMealTimes"];
+    UIViewController *controller = nil;
+    
+    switch (indexPath.row) {
+        case APHProfileSectionGlucoseLog:
+        {
+            UIStoryboard *sbGlucoseLog = [UIStoryboard storyboardWithName:@"APHOnboarding" bundle:[NSBundle mainBundle]];
+            APHGlucoseLevelsMealTimesViewController *glucoseController = [sbGlucoseLog instantiateViewControllerWithIdentifier:@"APHGlucoseMealTimes"];
+            
+            glucoseController.navigationController.navigationBar.topItem.title = NSLocalizedString(@"Glucose Log", @"");
+            glucoseController.hidesBottomBarWhenPushed = YES;
+            
+            glucoseController.isConfigureMode = YES;
+            
+            controller = glucoseController;
+            
+            [navigationController pushViewController:controller animated:YES];
+        }
+            break;
+            
+        default:
+        {
+            MFMailComposeViewController *feedbackController = [self feedback];
+            
+            self.weakNavController = navigationController;
+            
+            if (feedbackController) {
+                controller = feedbackController;
+                
+                [navigationController presentViewController:controller animated:YES completion:nil];
+            } else {
+                NSString *addEmailAccountMessage = NSLocalizedString(@"You don't have any email accounts set up. Please add an email account in Settings -> Mail, Contacts, Calendars", nil);
+                UIAlertController *alertView = [UIAlertController alertControllerWithTitle:NSLocalizedString(@"No Email Accounts", nil)
+                                                                                   message:addEmailAccountMessage
+                                                                            preferredStyle:UIAlertControllerStyleAlert];
+                
+                UIAlertAction* defaultAction = [UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault
```

Comment:
Yes. When the ```feedbackController``` was ```nil``` we did not see the alert being displayed, and the app crashed, because a nil view controller was being presented. Open to suggestion on how MFMailComposerViewController usage here could be leveraged handle ```nil``` situations.